src/renderer/store/playback/helpers.ts

```diff
@@ -0,0 +1,5 @@
+export interface PlaybackState {
+  playbackPlaying: boolean;
```

Comment:
This is good - can simplify the names a bit though because the `playback` bit is implied by the key of the object they're in

Something like

```
isPlaying: boolean;
playbackTime: number;
lastUpdated: Date;
```

or even change `playbackTime` to just `time`